test/did_you_mean/test_verbose_formatter.rb

```diff
@@ -17,6 +17,9 @@ def test_message
     assert_match <<~MESSAGE.strip, @error.message
       undefined method `zeor?' for 1:Integer
 
+          @error = assert_raise(NoMethodError){ 1.zeor? }
+                                                 ^^^^^^
+
           Did you mean? zero?
```

Comment:
In the future we may be able to make it cleaner. cc @mame

Reply:
FYI: I'm disabling did_you_mean when testing error_highlight.

https://github.com/ruby/error_highlight/blob/b79d679bbd5321dae819e344a6b843fdb29d14bd/test/test_error_highlight.rb#L13-L18